src/styles/blocks/linkbtn.scss

```diff
@@ -0,0 +1,25 @@
+$btn-height: 40px;
+$border-thickness: 1px;
+
+.linkbtn {
+  height: $btn-height;
+  display: block;
+  box-sizing: border-box;
+  border: $border-thickness solid #00acdc;
+  border-radius: 5px;
+  background-color: #00acdc;
+
+  font-size: 14px;
+  font-weight: 700;
+  line-height: $btn-height - $border-thickness * 2;
```

Comment:
The line-height calculation is incorrect. The expression `$btn-height - $border-thickness * 2` will not be evaluated as expected in SCSS. Consider using `calc(#{$btn-height} - #{$border-thickness} * 2)` to ensure proper calculation.